broker/machinebroker/cmd/machinebroker/app/app.go

```diff
@@ -211,3 +212,10 @@ func runStreamingServer(ctx context.Context, setupLog, log logr.Logger, srv *ser
 	}
 	return nil
 }
+
+func newGetter() (*config.BrokerGetter, error) {
+	return config.NewBrokerGetter(config.GetterOptions{
+		Name:           "machinebroker",
+		NetworkContext: egressselector.ControlPlane.AsNetworkContext(),
+	})
+}
```

Comment:
Can we please follow the same pattern like in the `poollets`? 
Here we take the getter from `poollet/bucketpoollet/client/config/getter.go`.

Reply:
Done